### PR TITLE
npm and package management

### PR DESCRIPTION
**Pull request by paulvanbrenk**
first pass for the "npm and package management " docs

### CODE REVIEW

File: articles/nodejs/npm.md [hunk 1]
```diff
@@ -1,3 +1,70 @@
...
+![Install new npm package from solution explorer](../../images/node/solution-explorer-install-package.png)
+
+In the window you can search for a package, specify options, and install. 
+
+![Search npm package](../../images/node/search-package.png)
+
+* **Dependency type** - allows you to chose between Standard, Development, and Optional. 
+Standard indicates the package is a runtime dependency, where 
+Development indicates it's only required during development.
+* **Add to package.json** - Deprecated
```
Comment (amcasey) [inline articles/nodejs/npm.md:25]: What does it mean for this to be deprecated?

Comment (paulvanbrenk) [reply articles/nodejs/npm.md:25]: it doesn't do anything

Comment (amcasey) [reply articles/nodejs/npm.md:25]: Does that mean we can remove it?

Comment (paulvanbrenk) [reply articles/nodejs/npm.md:25]: Yes, it's on my list of enhancements to this window.

**Comment by billti** [comment]
Why does 1 of the 3 image files have the extension capialized?

Also, in npm.md, the 3 image files reference between lines 46 and 49 do not exist.

**Pull request merged**